templates/scenarios/js/m365-tab/.vscode/launch.json

```diff
@@ -164,7 +164,7 @@
             "configurations": [
                 "Attach to Frontend in Outlook (Edge)"
             ],
-            "preLaunchTask": "Start Teams App Locally & Install App",
+            "preLaunchTask": "Start Teams App Locally",
```

Comment:
Any change on `launchUrl` to use new title/app id?

Reply:
Will update it afterwards when SS is ready. This PR is mainly for preparing the demo.